docs/std-archive.md

```diff
@@ -0,0 +1,42 @@
+# archive
+
+`archive` library contains helper functions that related to outputting data into
+certain formats of archive.
```

Comment:
```suggestion
certain archive formats (tar, zip).
```

Reply:
I'm not sure but we will probably have more formats